content/docs/manuals/networking-internals/mqtt-authentication.md

```diff
@@ -0,0 +1,163 @@
+            ```
+        4. Export the keyfile password as an environment variable:
+            ```bash
+            export CLUSTER_KEYFILE_PASSWORD=<keyfile password>
+            ```
+    * **Cluster Service Manager (Client):**
+        1. Generate a client key:
+            ```bash
+            openssl genrsa -aes256 -out cluster_net.key 2048
+            ```
+        2. Generate a certificate signing request:
+            ```bash
+            openssl req -out cluster_net.csr -key cluster_net.key -new
+            ```
+            When prompted for the CN, enter `cluster_service_manager`
+        3. Send the CSR to the CA:
+            ```bash
+            openssl x509 -req -in cluster_net.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out cluster_net.crt -days <duration>
+            ```
+        4. Export the keyfile password as an environment variable:
+            ```bash
+            export CLUSTER_SERVICE_KEYFILE_PASSWORD=<keyfile password>
+            ```
+
+3. Deploy the cluster with the MQTTS override
+    ```bash
+    sudo -E docker compose -f docker-compose.yml -f override-mosquitto-auth.yml
+    ```
+
+### Configuring a Node
+<!--- Subject to change when NodeEngine and NetManager are demonized -->
+
+1. Copy the `ca.crt` and `ca.key` files to the worker node.
+2. Generate the certificates
+   1. Generate a client key:
+       ```bashopenssl genrsa -aes256 -out client.key 2048
+       ```
+   2. Generate a certificate signing request:
+       ```bash
+       openssl req -out client.csr -key client.key -new
+       ```
+       When prompted for the CN, enter the public IP of the machine
+   3. Send the CSR to the CA:
+       ```bash
+       openssl x509 -req -in client.csr -CA <path to ca file> -CAkey <path to ca key file> -CAcreateserial -out client.crt -days <duration>
+       ```
+   4. Decrypt the keyfile:
+        ```bash
+        openssl rsa -in client.key -out unencrypt_client.key
+        ```
+   5. Tell your OS to trust the certificate authority by placing the ca.crt file in the `/etc/ssl/certs/` directory
+3. Run the NodeEngine:
+    ```bash
+    sudo NodeEngine -n 0 -p 10100 -a <SYSTEM_MANAGER_URL> -c <path to client.crt> -k <path to unencrypt_client.key>
+    ```
+4. **(Optional)** Configure the NetManager:
+    1. Repeat step 2 for the NetManager
+    2. Edit the `/etc/netmanager/netcfg.json` file so that the `"MqttCert"` and `"MqttKey"` fields specify the path to the NetManager certificate and key files
+    3. Run the NetManager:
+        ```bash
+        sudo NetManager -p 10100
+        ```
+
+
+{{< callout context="note" title="Did you know?" icon="outline/rocket">}} The [Oakestra automation repository](https://github.com/oakestra/automation)
+contains many useful scripts such as ones for [creating MQTTS certificate files](https://github.com/oakestra/automation/tree/d43f701134fdf71e1206532883006e1937c38ef9/development_cluster_management/generate_mqtts_certificates). {{< /callout >}}
```

Comment:
The end is quite abrupt. It would be great to have 1 sentence or paragraph telling the user that e.g. with this/congrats now you have added MQTTS, or provide an example of how to verify that this is indeed working.